schema/revision.ttl

```diff
@@ -0,0 +1,123 @@
+
+#################################################################
+#    Annotation properties
+#################################################################
+
+###  http://www.w3.org/2004/02/skos/core#prefLabel
+skos:prefLabel rdf:type owl:AnnotationProperty .
+
+
+#################################################################
+#    Datatypes
+#################################################################
+
+###  http://www.w3.org/2001/XMLSchema#date
+xsd:date rdf:type rdfs:Datatype .
+
+
+#################################################################
+#    Object Properties
+#################################################################
+
+###  http://www.w3.org/ns/prov#used
+<http://www.w3.org/ns/prov#used> rdf:type owl:ObjectProperty ;
+                                 rdfs:domain <http://www.w3.org/ns/prov#Activity> ;
+                                 rdfs:range <http://www.w3.org/ns/prov#Entity> ;
+                                 rdfs:comment "Usage is the beginning of utilizing an entity by an activity. Before usage, the activity had not begun to utilize this entity and could not have been affected by the entity." .
+
+
+###  https://rdf.equinor.com/ontology/revision/isNewRevisionOf
+:isNewRevisionOf rdf:type owl:ObjectProperty ;
+                rdfs:subPropertyOf <http://www.w3.org/ns/prov#wasDerivedFrom> ;
+                rdfs:domain <https://rdf.equinor.com/ontology/revision/Revision> ;
+                rdfs:range <https://rdf.equinor.com/ontology/revision/Revision> .
+
+###  https://rdf.equinor.com/ontology/revision/describes
+:describes rdf:type owl:ObjectProperty ;
+                rdfs:domain <https://rdf.equinor.com/ontology/revision/Revision> .
+
+###  https://rdf.equinor.com/ontology/revision/containsRecord
+<https://rdf.equinor.com/ontology/revision/containsRecord> rdf:type owl:ObjectProperty ;
+                                                         rdfs:subPropertyOf <http://www.w3.org/ns/prov#hadMember> ;
+                                                         rdf:type owl:AsymmetricProperty ,
+                                                                  owl:IrreflexiveProperty ;
+                                                         rdfs:domain <https://rdf.equinor.com/ontology/revision/Revision> ;
+                                                         rdfs:range <https://rdf.equinor.com/ontology/record/Record> ;
+                                                         rdfs:comment "A revision contains one or more records. They represent the imutable state which the revision author(s) used for the revision process." .
+
+
+#################################################################
+#    Data properties
+#################################################################
+
+###  http://www.w3.org/ns/prov#endedAtTime
+<http://www.w3.org/ns/prov#endedAtTime> rdf:type owl:DatatypeProperty ;
```

Comment:
Do we use this? Remove?

Reply:
Good catch. Used previously by a different revision model